docs/tutorial/part-three/index.md

```diff
@@ -1,57 +1,57 @@
+आप शायद प्लगइन्स के विचार से परिचित हैं। कई सॉफ्टवेयर सिस्टम नई कार्यक्षमता जोड़ने या सॉफ़्टवेयर के मुख्य कामकाज को संशोधित करने के लिए कस्टम प्लगइन्स को जोड़ने का समर्थन करते हैं। Gatsby प्लगइन्स उसी तरह काम करते हैं।
 
-Community members (like you!) can contribute plugins (small amounts of JavaScript code) that others can then use when building Gatsby sites.
+समुदाय के सदस्य (आप की तरह!) प्लगइन्स (जावास्क्रिप्ट कोड की छोटी मात्रा) का योगदान कर सकते हैं जो अन्य लोग तब उपयोग कर सकते हैं जब Gatsby साइटों का निर्माण करते हैं।
 
-> There are already hundreds of plugins! Explore the Gatsby [Plugin Library](/plugins/).
+> पहले से ही सैकड़ों प्लगइन्स हैं! गैट्सबी [प्लगइन लाइब्रेरी] (/pluginw/) का अन्वेषण करें।
 
-Our goal with plugins is to make them straightforward to install and use. You will likely be using plugins in almost every Gatsby site you build. While working through the rest of the tutorial you’ll have many opportunities to practice installing and using plugins.
+प्लगइन्स के साथ हमारा लक्ष्य उन्हें स्थापित करने और उपयोग करने के लिए सीधा करना है। आप संभवतः आपके द्वारा बनाए गए लगभग हर गैट्सबी साइट में प्लगइन्स का उपयोग कर रहे होंगे। बाकी ट्यूटोरियल के माध्यम से काम करते समय आपके पास प्लगइन्स को स्थापित करने और उपयोग करने के कई अवसर हैं।
 
-For an initial introduction to using plugins, we'll install and implement the Gatsby plugin for Typography.js.
+प्लगइन्स का उपयोग करने के लिए प्रारंभिक परिचय के लिए, हम टाइपोग्राफी के लिए Gatsby प्लगइन को स्थापित और कार्यान्वित करेंगे।
 
-[Typography.js](https://kyleamathews.github.io/typography.js/) is a JavaScript library which generates global base styles for your site's typography. The library has a [corresponding Gatsby plugin](/packages/gatsby-plugin-typography/) to streamline using it in a Gatsby site.
+[टाइपोग्राफी.जेएस] (https://kyleamathews.github.io/typography.js/) एक जावास्क्रिप्ट लाइब्रेरी है जो आपकी साइट की टाइपोग्राफी के लिए वैश्विक आधार शैली उत्पन्न करती है। लाइब्रेरी में एक Gatsby साइट का उपयोग करके इसे स्ट्रीमलाइन करने के लिए [संबंधित गैट्सबी प्लगइन] (/ संकुल / gatsby- प्लगइन-टाइपोग्राफी /) है।
 
-### ✋ Create a new Gatsby site
+### ✋ एक नई गैट्सबी साइट बनाएँ
 
-As we mentioned in [part two](/tutorial/part-two/), at this point it's probably a good idea to close the terminal window(s) and project files from previous parts of the tutorial, to keep things clean on your desktop. Then open a new terminal window and run the following commands to create a new Gatsby site in a directory called `tutorial-part-three` and then move to this new directory:
+जैसा कि हमने उल्लेख किया है [भाग दो] (/ ट्यूटोरियल / भाग-दो /) में, इस बिंदु पर टर्मिनल विंडो (एस) को बंद करना और ट्यूटोरियल के पिछले हिस्सों से परियोजना फाइलों को अपने काम पर रखने के लिए शायद एक अच्छा विचार है। डेस्कटॉप। फिर एक नई टर्मिनल विंडो खोलें और `ट्यूटोरियल-पार्ट-थ्री` नामक एक निर्देशिका में एक नई गैट्सबी साइट बनाने के लिए निम्न कमांड चलाएं और फिर इस नई निर्देशिका में जाएं:
```

Comment:
```
[भाग दो] (/ ट्यूटोरियल / भाग-दो /
```
=>
```
[भाग दो](/ट्यूटोरियल/भाग-दो/)
```

Reply:
We should not translate the links

`/ट्यूटोरियल/भाग-दो/` => `/tutorial/part-two/`

sorry missed this previously